src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/BasicScenarioTests.cs

```diff
@@ -40,5 +42,31 @@ public static IEnumerable<TestScenario> GetScenarios()
             yield return new(nameof(BasicScenarioTests), language, DotNetTemplate.NUnit,    DotNetActions.Test);
             yield return new(nameof(BasicScenarioTests), language, DotNetTemplate.MSTest,   DotNetActions.Test);
         }
+
+        yield return new(nameof(BasicScenarioTests), DotNetLanguage.CSharp, DotNetTemplate.WebApp, DotNetActions.PublishSelfContained, VerifyRuntimePacksForSelfContained);
```

Comment:
All web scenarios should be contained in the WebScenarioTests, not the BasicScenarioTests.

Reply:
My bad, I forgot to move it.

Thanks for pointing it out, moved